### PR TITLE
LRDOCS-6512 Created Documentation of Page Fragments Creation for 7.2

### PR DESCRIPTION
**Pull request by stephenkostas**
https://issues.liferay.com/browse/LRDOCS-6512

### CODE REVIEW

**Comment by sez11a** [comment]
Just started reviewing :)

File: discover/portal/articles/XX-segmentation-and-personalization/02-creating-segments.markdown [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Creating User Segments
```
Comment (sez11a) [inline discover/portal/articles/XX-segmentation-and-personalization/02-creating-segments.markdown]: What is this file? There's already a file in the proper folder (/user)....

File: discover/portal/articles/XX-segmentation-and-personalization/03-segment-criteria-reference.markdown [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# Defining Segmentation Criteria
```
Comment (sez11a) [inline discover/portal/articles/XX-segmentation-and-personalization/03-segment-criteria-reference.markdown]: Does this file belong here? There's already a folder for this in the proper place (/user).

**Comment by sez11a** [comment]
@stephenkostas There seem to be some extra files in here. I deleted one of them (a .java file from DocUtils). The other two look like they're left over from some file move, because they're in /discover/portal, which doesn't exist in the master branch. Please fix.

**Comment by stephenkostas** [comment]
Fixed title, ticket reference, and removed vestigial files.

**Comment by sez11a** [comment]
Just started reviewing :)

**Comment by sez11a** [comment]
Superseded by https://github.com/stephenkostas/liferay-docs/pull/71

**Pull request closed without merging**